crates/bevy_core_pipeline/src/core_3d/mod.rs

```diff
@@ -278,11 +297,15 @@ pub fn prepare_core_3d_depth_textures(
     >,
 ) {
     let mut textures = HashMap::default();
-    for (entity, camera) in &views_3d {
+    for (entity, camera, prepass_settings) in &views_3d {
         if let Some(physical_target_size) = camera.physical_target_size {
             let cached_texture = textures
                 .entry(camera.target.clone())
                 .or_insert_with(|| {
+                    let mut usage = TextureUsages::RENDER_ATTACHMENT;
```

Comment:
The raw logic of this segment is hard to follow: can we refactor this to be simpler? If not, comments will help.

Reply:
I tried improving it a little bit but flattening it a bit, but honestly I'm not sure what's complicated here. It's pretty much just getting a texture from a cache if it doesn't exist.